src/NLog.WindowsIdentity/NLog.WindowsIdentity.csproj

```diff
@@ -73,6 +73,10 @@ ImpersonatingWrapper Target Docs:
     <Title>NLog.WindowsIdentity for NetStandard 2.0</Title>
   </PropertyGroup>
 
+  <PropertyGroup Condition=" '$(TargetFramework)' == 'net6.0' ">
+    <Title>NLog.WindowsIdentity for Net 6.0</Title>
```

Comment:
Can it become `NLog.WindowsIdentity for .NET 6.0`

Reply:
Sure